docs/programming/MPI-C.md

```diff
@@ -227,7 +225,7 @@ Hello World from process 1 of 4
 Hello World from process 0 of 4
 ```
 
-Ref: [http://www.dartmouth.edu/~rc/classes/intro_mpi/hello_world_ex.html](http://www.dartmouth.edu/~rc/classes/intro_mpi/hello_world_ex.html)
+Ref (Dead Link): [http://www.dartmouth.edu/~rc/classes/intro_mpi/hello_world_ex.html](http://www.dartmouth.edu/~rc/classes/intro_mpi/hello_world_ex.html)
```

Comment:
If this is a dead link, please remove it.

Reply:
Shouldn't we still be citing our source?

Reply:
If we want to give credit, we should state that the material was inspired by an intro to MPI course taught at Dartmouth. Deadlinks are useless. The other option is to find a link that is not dead where this material was pulled from.